app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java

```diff
@@ -150,8 +150,12 @@ public boolean onQueryTextSubmit(String s) {
       @Override
       public boolean onQueryTextChange(String s) {
         if (s.equals("")) {
+          View item = findViewById(R.id.menu_searchintext);
```

Comment:
I don't think these changes are relevant to this issue.

Reply:
Sorry,
Reverting these changes here.I have created another pull request regarding the issue.